### ISSUE TITLE
Thank you. and some questions.

### ISSUE THREAD

**Issue by Nate-mycroft** [Hasinator7/nature-sound-skill#1]
@Hasinator7 Thank you so much for creating this.  This is something our founder had requested in the past.  

A couple of questions:  
1.  Is there a voice command you have set up to voice install the skill?
2.  If I have some other sound files, can we merge them in?

Thanks,
Nate Tomasi
nate.tomasi@mycroft.ai
COO Mycroft AI

**Comment by Hasinator7** [comment]
Nate, 
1. Does it need the requirements.py and requirements.sh for voice installation? If that is so, do empty requirements work?
2. I'd love to expand the soundbase. I had quite a few for testing, but they were not under Creative Commons and that's what you need for a project like this, right?

**Comment by Nate-mycroft** [comment]
1.  Sounds like you submitted the PR, and I will push the Skills team to accept.  Sounds like there are a few things that need to be updated, and you should be good.  They will comment soon for you.   
2.  Probably, but if someone requests to take it out we can.   I wouldn't worry about it too much though.   Here is the contest that our founder sent out, with some audio.  Take a look if you want to merge any of those in.   If you can get some of those in, and the skill is approved to use, close the contest down.  

Nate

**Comment by Hasinator7** [comment]
Did you mean to  put a link in your comment?

**Comment by Nate-mycroft** [comment]
Yes, sorry:  https://community.mycroft.ai/t/white-noise-contest/2063
<https://community.mycroft.ai/t/white-noise-contest/2063>

Nate Tomasi
Mycroft.AI <https://mycroft.ai/> | Chief Operating Officer | 573.268.2667
4190 Willmar Drive, Palo Alto, CA, 94306
<https://maps.google.com/?q=4190+Willmar+Drive,+Palo+Alto,+CA,+94306&entry=gmail&source=g>
